### ISSUE TITLE
Please include simple  script example in README.md

### ISSUE THREAD

**Issue by reikred** [termux/termux-widget#86]
The README.md ought to include a small hello world or equivalent sample script, and in particular it should include that the shebang should have the form

#!/data/data/com.termux/files/usr/bin/bash

or similar for other shells. There is too much guesswork that takes time. Just a very simple example will do.

**Comment by Grimler91** [comment]
> First thing: The README.md ought to include a small "hello world" or similar sample script, and in particular it should include that the shebang should have the form

Sure, we can add that

> but when I start the termux:widget, it does not appear as a clickable button

Shortcurt needs to be added to the home screen as a widget, exactly how to do that vary between devices, but typically you long press home screen, press widgets, and then scroll down to the "termux" section:

![widget_1](https://user-images.githubusercontent.com/18900601/188382144-2e123c7b-276f-434a-96d6-6e38095d4764.png)

![widget_2](https://user-images.githubusercontent.com/18900601/188382160-aee49163-6634-4d07-ba5c-ce09caf33f2f.png)

**Comment by reikred** [comment]
Thank you, I picked the big widget and moved it to an empty screen. It seems to work now, but my script runs again and again although ai invoked it only once. I'd better write up another issue about that.

**Comment by agnostic-apollo** [comment]
Widget are not meant to be moved into folders, you can move the shortcut instead.

**Comment by reikred** [comment]
Thanks, I also read somewhere that alternative launcher apps such as yandex launcher supports widgets in folder, but I have not tested yet.

**Comment by agnostic-apollo** [comment]
It does apparently.

>Smart folders: Give every folder its own screen with widgets and a background.

https://play.google.com/store/apps/details?id=com.yandex.launcher&hl=en&gl=US

It is "technically" possible, but not something Termux:Widget would need to do anything about or support since widget creation is handled by the launcher app, unless there are some UI issues that need fixing in folder mode that are being caused by Termux:Widget.
